src/main/resources/edu/cornell/kfs/coa/businessobject/datadictionary/HigherEducationFunction.xml

```diff
@@ -1,51 +1,50 @@
-	<bean id="HigherEducationFunction" parent="HigherEducationFunction-parentBean">
-		<property name="attributes">
-			<list merge="true">
-				<ref
-					bean="HigherEducationFunction-extension-financialHigherEdFunctionDescription" />
-			</list>
-		</property>
-	</bean>
+    <bean id="HigherEducationFunction" parent="HigherEducationFunction-parentBean">
+        <property name="attributes">
+            <list merge="true">
+                <ref bean="HigherEducationFunction-extension-financialHigherEdFunctionDescription" />
+            </list>
+        </property>
+    </bean>
 
-	<!-- Attribute Definitions -->
+    <!-- Attribute Definitions -->
 
-	<bean
-		id="HigherEducationFunction-extension-financialHigherEdFunctionDescription"
-		parent="HigherEducationFunction-extension-financialHigherEdFunctionDescription-parentBean" />
-	<bean
-		id="HigherEducationFunction-extension-financialHigherEdFunctionDescription-parentBean"
-		abstract="true"
-		parent="HigherEducationFunctionExtendedAttribute-financialHigherEdFunctionDescription"
-		p:name="extension.financialHigherEdFunctionDescription" />
+    <bean
+        id="HigherEducationFunction-extension-financialHigherEdFunctionDescription"
+        parent="HigherEducationFunction-extension-financialHigherEdFunctionDescription-parentBean" />
+    <bean
+        id="HigherEducationFunction-extension-financialHigherEdFunctionDescription-parentBean"
+        abstract="true"
+        parent="HigherEducationFunctionExtendedAttribute-financialHigherEdFunctionDescription"
+        p:name="extension.financialHigherEdFunctionDescription" />
 
-	<!-- Business Object Inquiry Definition -->
+    <!-- Business Object Inquiry Definition -->
 
-	<bean id="HigherEducationFunction-inquiryDefinition-override"
-		parent="DataDictionaryBeanOverride">
-		<property name="beanName" value="HigherEducationFunction-inquiryDefinition" />
-		<property name="fieldOverrides">
-			<list>
-				<bean parent="FieldOverrideForListElementInsert">
-					<property name="propertyName" value="inquirySections[0].inquiryFields" />
-					<property name="propertyNameForElementCompare" value="attributeName" />
-					<property name="element">
-						<bean parent="FieldDefinition" p:attributeName="financialHigherEdFunctionNm" />
-					</property>
-					<property name="insertAfter">
-						<list>
-							<bean parent="FieldDefinition"
-								p:attributeName="extension.financialHigherEdFunctionDescription" />
-						</list>
-					</property>
-				</bean>
-			</list>
-		</property>
-	</bean>
+    <bean id="HigherEducationFunction-inquiryDefinition-override"
+        parent="DataDictionaryBeanOverride">
+        <property name="beanName" value="HigherEducationFunction-inquiryDefinition" />
+        <property name="fieldOverrides">
+            <list>
+                <bean parent="FieldOverrideForListElementInsert">
+                    <property name="propertyName" value="inquirySections[0].inquiryFields" />

```

Comment:
Could you identify an existing example of what you are referring to?
Changes I am trying to make locally to address your concern are not working as expected.

Reply:
Given where the override needs to be, you could add a new "FieldOverrideForListElementInsert" bean into the "fieldOverrides" list property on the "HigherEducationFunction-inquiryDefinition-override" bean. KualiCo's new inquiry setup is very recent, and I don't think we have an existing example of a field-override bean for that. However, it looks like such a bean should have a setup like the following, to perform the equivalent insert from our existing bean above:

```xml
<bean parent="FieldOverrideForListElementInsert"
      p:propertyName="sections[0].fields"
      p:propertyNameForElementCompare="name"
      p:element-ref="HigherEducationFunction-financialHigherEdFunctionNm"
      p:insertAfter-ref="HigherEducationFunction-extension-financialHigherEdFunctionDescription"/>
```

Note that the "element" and "insertAfter" properties might need to be replaced with child beans instead of bean refs, if the inquiry or other related screens run into unexpected issues.

Reply:
As another alternative, you could add a DataDictionaryBeanOverride bean that operates on the "HigherEducationFunction-sectionDefinition" bean instead, and then configure the field override to insert the element into the "fields" list. Perhaps this way would be a little clearer, so that you can simplify the list property path in the field-override bean.